### ISSUE TITLE
Not loading large images

### ISSUE THREAD

**Issue by andrewsilberfeld** [BIOP/ijp-imagetoatlas#115]
Hello,

When I import a QuPath file consisting of 20 downsampled tiff images (about 5-mb each), all the images appear in ABBA. Yet when I import a QuPath file consisting of 20 ome.tif files at full scale (about 18 to 128 mb each; total size 1.13 GB), I only get the last file in the QuPath project. Any idea how to fix this? I'd like to count cells on the full-size image in QuPath after registration but I can't bring them into ABBA.

Best,
Andrew

**Comment by NicoKiaru** [comment]
Hi Andrew, 

Did you select the Bio-Formats image server ? Otherwise can you share your images ? There should not be any issue with big OME-TIFFs

**Comment by andrewsilberfeld** [comment]
Yeah Bio-Formats server was selected. here's the import settings: 
<img width="960" alt="Import settings for ome-tiff error" src="https://user-images.githubusercontent.com/62805456/144684790-8009cb1e-3688-4b49-8d1b-03c09962dcf1.png">

and here is the result after importing to abba. It is actually more common that I get only the first two images from my list. It is less common that I get only the last image on the list, but both are obviously not correct.

**Comment by NicoKiaru** [comment]
I can't reproduce the issue unfortunately, all images open without any issue on my side (with a "regular" win 10 laptop), but you may try two things:

When importing your images, uncheck 'use bioformats cache block size' and choose something like 512x512 or 1024x1024:

![image](https://user-images.githubusercontent.com/20223054/144715980-60400706-c8c4-4e60-9d89-2220830a297a.png)

That may work. 

But that's probably not the best way. Your real problem is that your full resolution ome.tiff images are not pyramidal:

![image](https://user-images.githubusercontent.com/20223054/144716025-1718f945-6f4b-4880-9571-8f9236d149fd.png)

In 'pyramid', there is 'NO' written. So the full resolution image has to be directly loaded. If you convert your files to pyramidal ome-tiff (see for instance https://qupath.readthedocs.io/en/stable/docs/advanced/exporting_images.html ), 

![image](https://user-images.githubusercontent.com/20223054/144716256-f238c5e7-38c5-4660-a8f7-b399b7d445e1.png)

everything (QuPath and ABBA) will be much more responsive. And by the way, with pyramidal images, you don't need to generate yourself downsampled versions of your images. Both QuPath and ABBA will use the contained low resolution image when  needed.

**Comment by andrewsilberfeld** [comment]
Thanks! Okay the pyramidal problem solved the issue (and dechecking the bioformats cache block size with 512x512 or 1024x1024 did not resolve it). 

I re-exported my files using the QuPath File/Export Images/OME TIFF option, made a new QuPath project using those files, and opened them in ABBA. All of them loaded now. 

Thank you!

For future users: this probably happened because of how I processed the images. I originally didn't want to segment my tissue on the slide scanner as I thought I would be faster on Fiji so I imaged the slide as 1 big file and cropped each section into its own file using Fiji. I used the ome-tiff option when I saved in fiji but this clearly was inadequate.
